datahub-graphql-core/src/main/resources/timeline.graphql

```diff
@@ -0,0 +1,189 @@
+extend type Query {
+    """
+    Get list of schema blame
+    """
+    getSchemaBlame(input: GetSchemaBlameInput!): GetSchemaBlameResult
+}
+
+type TimelineConfig {
+    """
+    Whether the timeline feature is enabled.
+    """
+    enabled: Boolean!
+}
+
+"""
+Enum of SemanticVersionChange
+"""
+enum SemanticVersionChangeType {
+    """
```

Comment:
These comments do not inform me of the situations in which i can expect one change type vs another. Can we remove any of these? Once we add, will be challenging to remove

Reply:
I don't think so because this is just a translation of what exists on the Timeline API.

Reply:
@RyanHolstien are all of these actively used? Seems not ideal to pollute public API if only a few are used

Reply:
I removed it since I think I agree, we don't need it here.